src/include/schema.h

```diff
@@ -296,7 +296,9 @@ struct __wt_table {
 		F_CLR(session, WT_SESSION_LOCKED_CHECKPOINT);		\
 		__wt_spin_unlock(session, &__conn->checkpoint_lock);	\
 	}								\
+	__wt_yield();							\
 	op;								\
+	__wt_yield();							\
```

Comment:
This change is not related to the main fix here but was something I noticed during testing -- yielding here avoid operations like `verify` getting "stuck" temporarily while trying to close an LSM tree because the closing thread was doing 1000 yields while holding locks.